test/components/CalendarMonthGrid_spec.jsx

```diff
@@ -51,4 +52,24 @@ describe('CalendarMonthGrid', () => {
       expect(wrapper.prop('style')[key]).to.equal(transformStyles[key]);
     });
   });
+
+  it('does not generate duplicate months', () => {
+    const initialMonth = moment();
+    const wrapper = shallow(<CalendarMonthGrid numberOfMonths={12} initialMonth={initialMonth} />);
+
+    wrapper.instance().componentWillReceiveProps({
+      initialMonth,
+      numberOfMonths: 24,
+    });
+
+    const months = wrapper.state().months;
+
+    const collisions = months
+      .map(m => m.format('YYYY-MM'))
+      .reduce((acc, m) => Object.assign(
+        {}, acc, { [m]: true }
```

Comment:
if the invocation parens and every argument can't fit on one line, then each argument must be on a line by itself.